CHANGELOG.rst

```diff
@@ -21,6 +21,8 @@ Changed
 -------
 - new event broker class: ``SQLProducer``. This event broker is now used when running locally with
   Rasa X
+- API requests are not longer logged to ``rasa_core.log`` by default in order to avoid
+  problems when running on OpenShift
```

Comment:
Maybe with a little migration here -- "to get logs to this file pass the `--log_file rasa_core.log` parameter or some other way to let people know that this setting has changed? A log upon server start or something? Just worried about people who rely on this default and then don't realize when it stops logging

Reply:
added it to the changelog, not worth adding it to the migration guide imo